src/useRafLoop.ts

```diff
@@ -47,8 +46,7 @@ export default function useRafLoop(
     }
 
     return result[0];
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  }, [initiallyActive, result]);
```

Comment:
Does this solve some actual bug or does it just fix linter errors? I'm worried that it might introduce regressions.

Reply:
it just fix linter errors